src/InvokePlaster.ps1

```diff
@@ -1,4 +1,4 @@
-<#
+﻿<#
```

Comment:
I did, yes. I did a little investigation, it's the HELPERS art bit, the characters are non-ASCII code points.